### PR TITLE
Alusine-Node.js homework week2

### PR DESCRIPTION
**Pull request by alquinex**

### CODE REVIEW

File: week2/homework/src/add.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+// Adds a To-do
+var addTodo = title => {
+  var todos = [];
+  var todo = {
+    title,
+  };
+
+  try {
+    var todosString = fs.readFileSync('todos-data.json');
+    todos = JSON.parse(todosString);
```
Comment (gajduk) [inline week2/homework/src/add.js]: using JSON 👍

File: week2/homework/src/add.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  } catch (e) {}
+
+  var duplicatetodos = todos.filter(todo => todo.title === title);
+
+  if (duplicatetodos.length === 0) {
+    todos.push(todo);
+    fs.writeFileSync('todos-data.json', JSON.stringify(todos));
```
Comment (gajduk) [inline week2/homework/src/add.js]: Can this throw an error?

File: week2/homework/src/deletes.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+let deleteTodo = title => {
+  let todos = fetchTodos();
+  let filteredtodos = todos.filter(todo => todo.title !== title);
+  saveTodos(filteredtodos);
+
+  return todos.length !== filteredtodos.length;
+};
+
+// Utility Functions
+let fetchTodos = () => {
```
Comment (gajduk) [inline week2/homework/src/deletes.js]: nice that you made this a function. However it is not used anywhere else :(

File: week2/homework/src/add.js (hunk 1 above)
Comment (alquinex) [reply week2/homework/src/add.js]: Hi @gajduk, thanks for the feedback and for your time. The reset command has been added.

**Comment by gajduk** [comment]
Looks good

**Comment by fedefu** [comment]
Thanks @gajduk for all the feedback! Really appreciate it :)

**Pull request closed without merging**